src/main/java/com/blackduck/integration/blackduck/bdio2/Bdio2UploadJob.java

```diff
@@ -75,6 +75,9 @@ public void attemptJob() throws IntegrationException {
                 uploadUrl = new HttpUrl(headerResponse.getHeaderValue("location"));
                 scanId = parseScanIdFromUploadUrl(uploadUrl.string());
             } else {
+                if(!blackDuckUrl.string().endsWith("/")) {
```

Comment:
Since blackDuckUrl is nullable, do you need to perform a not null check in this block?